src/Common/FitOnFhir.Common.Tests/AuthStateServiceTests.cs

```diff
@@ -3,10 +3,12 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
+using System;
 using System.IdentityModel.Tokens.Jwt;
 using Azure;
 using Azure.Storage.Blobs;
 using Azure.Storage.Blobs.Models;
+using EnsureThat;
 using Microsoft.Extensions.Logging;
```

Comment:
System and Ensure that are not needed and are causing a compile error on my machine.

Reply:
I wish whatever checked that was more reliable.  I don't get those errors on my machine.

Reply:
I am going to import the global project settings from the workspace-platform repository and that should apply the same settings in your IDE.